core/src/err/mod.rs

```diff
@@ -779,6 +779,142 @@ pub enum Error {
 	/// The db is running without an available storage engine
 	#[error("The db is running without an available storage engine")]
 	MissingStorageEngine,
+
+	/// The requested analyzer already exists
+	#[error("The analyzer '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	AzAlreadyExists {
+		value: String,
+	},
+
+	/// The requested database already exists
+	#[error("The database '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	DbAlreadyExists {
+		value: String,
+	},
+
+	/// The requested event already exists
+	#[error("The event '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	EvAlreadyExists {
+		value: String,
+	},
+
+	/// The requested field already exists
+	#[error("The field '{value}' already exists")]
+	#[cfg(feature = "sql2")]
+	FdAlreadyExists {
```

Comment:
`FieldAlreadyExists`?

Reply:
Considering all related errors have the same naming scheme, I prefer the shorthand name